cymetric/timeseries.py

```diff
@@ -0,0 +1,321 @@
+import warning
+
+import pandas as pd
+import numpy as np
+
+try:
+    from pyne import data
+    import pyne.enrichment as enr
+    from pyne import nucname
+    HAVE_PYNE = True
+except ImportError:
+    HAVE_PYNE = False
+
+
+import cymetric as cym
+from cymetric import tools
+
+from cymetric.filter import get_transaction_df
+from cymetric.filter import get_transaction_nuc_df
+from cymetric.filter import get_transaction_activity_df
+from cymetric.filter import get_transaction_decayheat_df
+from cymetric.filter import get_inventory_df
+from cymetric.filter import get_inventory_activity_df
+from cymetric.filter import get_inventory_decayheat_df
+
+
+def get_transaction_timeseries(evaler, senders=(), receivers=(), commodities=(), nucs=()):
```

Comment:
I think this could be called `transactions()` and then above you would `from cymetric import filters` and call `filters.transactions()` here.